cogs/ahk/ahk.py

```diff
@@ -711,9 +711,10 @@ def check(inter: disnake.MessageInteraction):
 
         await thread.edit(applied_tags=added_tags)
 
+        tag_list = " and ".join((tag.emoji.name if tag.emoji else "") + tag.name for tag in added_tags)
         await message.edit(
             content=(
-                "Thanks for tagging your post!\n\nIf your issue gets solved, you can mark your post as solved by sending `/solved`"
+                "Thanks for tagging your post as " + tag_list + "!\n\nIf your issue gets solved, you can mark your post as solved by sending `/solved`"
```

Comment:
I don't think the added tags need to be specified in this message, at least make it say "a, b, c, and d" instead of "a and b and c and d"

Reply:
there can only be up to two tags, so this will make it a and b if there are two and a if there's one